conf/.env.example

```diff
@@ -19,7 +20,7 @@ CHECK_FOR_UPDATES=check_if_updates_are_available
 
 # EMAIL CONFIGURATION
 DEFAULT_FROM_EMAIL=tooljet@__DOMAIN__
-SMTP_DOMAIN=localhost
+SMTP_DOMAIN=__MAIN_DOMAIN__
 SMTP_PORT=25
```

Comment:
(maybe there are some options to specify the user/password to auth on the SMTP server, cf the user mail feature in 11.2 ... though it requires manifest v2)